### PR TITLE
RF: Make the tutorial data less special.

### PR DESCRIPTION
**Pull request by hanke**
- load_datadb_tutorial_data() -> load_tutorial_data()

  we should not care where it is coming from

- Makefile TUT_DIR now points directly to the 'data' directory

- load_tutorial_data() can now also load the two other flavors (1slice
  and 25mm) that we ship with the sources. This should make it possible
  to (at least) run a lot of code that used to require the 300MB
  download before. It could also be used to speed tests up be some
  factor.

- this is a move into the general direction of gh-245

### CODE REVIEW

**Comment by hanke** [comment]
Merging this is now a requirement for gh-307

File: mvpa2/datasets/sources/native.py [hunk 1]
```diff
@@ -51,15 +52,18 @@ def load_example_fmri_dataset(name='1slice', literal=False):
     return ds
 
 
-def load_datadb_tutorial_data(path=os.path.join(
```
Comment (yarikoptic) [inline mvpa2/datasets/sources/native.py]: couldn't we provide some deprecation path here?

Comment (hanke) [inline mvpa2/datasets/sources/native.py]: like this:

````
load_datadb_tutorial_data = load_tutorial_data
````

?

**Comment by hanke** [comment]
I am closing this one. It is merged into gh-307 and keep it open and separate is slowing things down....

**Pull request closed without merging**